Gemfile

```diff
@@ -49,6 +49,13 @@ group :development, :test do
   gem 'pry-rails'
   gem 'pry-byebug'
   gem 'pry-stack_explorer'
+  gem 'rubocop'
+  gem "onkcop", require: false
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `onkcop` should appear before `rubocop`.